editor/run/game_view_plugin.cpp

```diff
@@ -1074,6 +1116,27 @@ GameView::GameView(Ref<GameViewDebugger> p_debugger, EmbeddedProcessBase *p_embe
 	next_frame_button->set_accessibility_name(TTRC("Next Frame"));
 	next_frame_button->set_shortcut(ED_GET_SHORTCUT("editor/next_frame_embedded_project"));
 
+	speed_state_button = memnew(MenuButton);
+	main_menu_hbox->add_child(speed_state_button);
+	speed_state_button->set_text(TTRC("1.00x"));
+	speed_state_button->set_theme_type_variation("FlatButton");
+	speed_state_button->set_tooltip_text(TTRC("Change the game speed."));
+	speed_state_button->set_accessibility_name(TTRC("Speed State"));
+	speed_state_button->set_custom_minimum_size(Vector2(48, 0));
+
+	PopupMenu *menu = speed_state_button->get_popup();
+	menu->connect(SceneStringName(id_pressed), callable_mp(this, &GameView::_speed_state_menu_pressed));
+	for (float f : time_scale_range) {
+		menu->add_item(TTRC(vformat("%0.2fx", f)));
```

Comment:
```suggestion
		menu->add_item(TTRC(vformat(U"%s×", String::num(f))));
```

Reply:
@DexterFstone same as above here - no need to mark numbers for translation (remove `TTRC`).